src/main/groovy/com/deploygate/gradle/plugins/dsl/syntax/DeploymentSyntax.groovy

```diff
@@ -8,8 +8,6 @@ interface DeploymentSyntax {
 
     void setMessage(@Nullable String message)
 
-    void setVisibility(@Nullable String visibility)
```

Comment:
We should never allow breaking changes to API without any deprecation phase. Please make this deprecated and print warnings.